index.ts

```diff
@@ -113,4 +114,6 @@ strip.first = (input, options) => {
  * @api public
  */
 
-strip.parse = parse;
+ strip.parse = parse;
+
+ export default strip
```

Comment:
So, this is something of a pain.  I'm not aware of a way to get TypeScript to export what we current have.  i.e. if we want to do the following

```
const strip = require('strip-comments');
strip('// comment');
```

this fails, because `strip` is an object `{ default: { parse, first, line, block } }`

The only way I can see to achieve this is via a bundler (rollup, for example). Any idea if we can achieve that in a less involved way?

Reply:
I was confused by this as well tbh, but I think it actually works as is. If you call strip() it runs the default function but strip is also an object that holds a reference to the other functions like block and line. Seems TS shows the correct intellisense for this as well.

Reply:
It works perfectly in some contexts, but not all.  If the module importing `index.ts` is also compiled by TS, all is well because the importing module should look like this:

```
var index_1 = __importDefault(require("../index"));
```
for the import and
```
index_1.default;
```
when it's used. If you require `index.js` directly node does not automatically pull out the default function (which is the strip function with all it's properties).

We don't need to worry about it too much in this PR.I was just hoping there was a secret tsconfig property that dealt with this, but I guess not.